builtins/packagedcode_msitools-linux/Makefile

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
We now use  a simplified header. Can you check in nearby plugins for inspiration? (I wish we could use the skeleton to merge in subdirs... may be this is possible too? )

Reply:
I've updated the file header.